apps/cli-daemon/src/app/abstract/ng-command-executor.base.ts

```diff
@@ -0,0 +1,22 @@
+import { spawnSync, SpawnSyncOptionsWithBufferEncoding } from 'child_process';
+import { resolve as pathResolve } from 'path';
+
+import { ExecResult } from '../generators/dto';
+import { SessionService } from '../session/session.service';
+
+export abstract class NgCommandExecutorBase {
```

Comment:
I'd prefer to avoid extending services and controllers unless it's really necessary in the favor of composition.

Reply:
👍